apps.json

```diff
@@ -551,6 +551,12 @@
         ],
         "url": "https://github.com/YunoHost-Apps/distbin_ynh"
     },
+    "django-for-runners": {
+        "branch": "master",
```

Comment:
You should add a master branch adn make it the default branch

Reply:
Ah sorry. Created: https://github.com/YunoHost-Apps/django-for-runners_ynh/tree/master